IBM/ibm-spectrum-scale-csi
# csv update

Changes : 
- New Logo 
 - Updated Description

Test Suggestion - 
1. Check preview on https://operatorhub.io/preview
2. Check Operator listing on Various platform and Arch
